### DISCUSSION TITLE
Supported Home Assistant version

### DISCUSSION THREAD

**Discussion by pajdewaal** [robbinjanssen/home-assistant-omnik-inverter#281] [Q&A] [answered]
I've been using the integration with Home Assistant 2023.4 for a while now. However I would like to update HASS to a more recent version (so I can update Zigbee2MQTT). So preferrably 2023.8 +. However, when i load a newer version, the Omnik Integration fails to load. Tried everything, updating HACS, Reinstalling, re-configuring also keeps failing with any HASS version 2023.4+

Is this related to HASS version not being supported or perhaps something else I'm doing wrong?

`2024-01-31 09:33:13.437 DEBUG (MainThread) [custom_components.omnik_invertor] Finished fetching omnik_inverter data in 0.209 seconds (success: True)
2024-01-31 09:33:13.437 ERROR (MainThread) [homeassistant.config_entries] Error setting up entry omnik_omvormer for omnik_inverter
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 406, in async_setup
    result = await component.async_setup_entry(hass, self)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/omnik_invertor/__init__.py", line 46, in async_setup_entry
    hass.config_entries.async_setup_platforms(entry, PLATFORMS)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'ConfigEntries' object has no attribute 'async_setup_platforms'
`

**Comment by klaasnicolaas** [comment, marked as answer]
You are using an old version of the integration

**Reply by pajdewaal** [reply]
HACS updated it to 2.6.4. Is that not the latest version?

**Reply by klaasnicolaas** [reply]
That's latest, but we solved the exception a long time ago. So the fact that you get that error tells me that you are using an older one.

**Reply by pajdewaal** [reply]
thanks. Appears that in custom_components I had 2 folders named omnik. HACS reported it at latest version, but indeed it kept using the older version. Complete new HASS, HACS and integration did the trick.
